src/dags/precariobelasting.py

```diff
@@ -75,39 +76,39 @@ def clean_data(file_name):
     # 2. create download temp directory to store the data
     mk_tmp_dir = BashOperator(task_id="mk_tmp_dir", bash_command=f"mkdir -p {tmp_dir}")
 
-    # 3. download the data into temp directory
+    # 3. Download data
     download_data = [
-        HttpFetchOperator(
+        SwiftOperator(
             task_id=f"download_{file_name}",
-            endpoint=url,
-            http_conn_id="airflow_home_conn_id",
-            tmp_file=f"{tmp_dir}/{file_name}.json",
-            output_type="file",
+            swift_conn_id="objectstore_dataservices",
+            container="Dataservices",
+            object_id=url,
+            output_path=f"{tmp_dir}/{url}",
         )
         for file_name, url in data_endpoints.items()
     ]
 
     # 4. Cleanse the downloaded data (remove the space hyphen characters)
-    clean_data = [
+    clean_up_data = [
         PythonOperator(
             task_id=f"clean_data_{file_name}",
             python_callable=clean_data,
-            op_args=[f"{tmp_dir}/{file_name}.json"],
+            op_args=[f"{tmp_dir}/{url}"],
         )
-        for file_name in data_endpoints.keys()
+        for file_name, url in data_endpoints.items()
     ]
 
     # 5.create the SQL for creating the table using ORG2OGR PGDump
     extract_geojsons = [
         BashOperator(
             task_id=f"extract_geojson_{file_name}",
-            bash_command="ogr2ogr -f 'PGDump' "
-            "-t_srs EPSG:28992 "
+            bash_command=f"ogr2ogr -f 'PGDump' "
+            f"-t_srs EPSG:28992 "
```

Comment:
Volgens mij heeft Guido zouist veel onnodig f-strings weggehaald. Het lijkt erop of ze hiermee juiste weer terugkomen?
(De f-string is alleen nodig als er een variabele in de string wordt gebruikt).

Reply:
Of, het kan ook komen dat de f-strings juist weer terugkomen om de laatste master niet is gepulled?